### PR TITLE
docs: Preserve output, clean eval notebooks

### LINKED ISSUE #1559: CI: Preserve the output of the eval notebooks
We would like to have eval notebooks that contain a clean output of the results without containing secret openAI keys in the metadata. The notebooks need to be a bit cleaner so it's not polluted with output.

Acceptance criteria
- notebook metadata cleared
- OpenAI keys etc are not embedded
- cleanup the notebooks to not output massive progress bars etc.

### PR DESCRIPTION
**Pull request by pbadhe**
Resolves #1559 
+ Output 
- Removed metadata and execution count

### CODE REVIEW

**Comment by axiomofjoy** [comment]
Hey @pbadhe, thanks for picking up this ticket. In addition to executing the notebooks to include input, we also need to alter the `hatch` notebook command for cleaning the notebook to exclude notebooks inside the `evals` directory.

https://github.com/Arize-ai/phoenix/blob/50e765bb98dc84c540401dd14135104ec6582678/pyproject.toml#L199

This command is causing the current pipeline failure: https://github.com/Arize-ai/phoenix/blob/50e765bb98dc84c540401dd14135104ec6582678/.github/workflows/python-CI.yml#L51

File: pyproject.toml [hunk 1]
```diff
@@ -197,7 +197,7 @@ fix = [
 
 [tool.hatch.envs.notebooks.scripts]
 clean = [
-  "jupyter nbconvert --ClearOutputPreprocessor.enabled=True --ClearMetadataPreprocessor.enabled=True --inplace **/*.ipynb **/**/*.ipynb",
+  "jupyter nbconvert --ClearOutputPreprocessor.enabled=True --ClearMetadataPreprocessor.enabled=True --inplace **/*.ipynb **/internal/*.ipynb **/tracing/*.ipynb **/dolly-pythia-fine-tuned/*.ipynb",
```
Comment (pbadhe) [inline pyproject.toml:200]: Had to do this as I couldn't find a negation(!) wildcard (for all *.ipynb except evals/*) that works in both windows and unix at the same time.

Comment (axiomofjoy) [reply pyproject.toml:200]: Is there a shell command that works?

Comment (pbadhe) [reply pyproject.toml:200]: This one does `find . -name "*.ipynb" -not -path "./tutorials/evals/*" -exec jupyter nbconvert --ClearOutputPreprocessor.enabled=True --ClearMetadataPreprocessor.enabled=True --inplace {} \;`

**Review by axiomofjoy — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (axiomofjoy) [reply pyproject.toml:200]: Looks good.

**Comment by mikeldking** [comment]
We missed the pre-commit hook (which I think is sorta a problem that we have two checks). I'll fix that

**Pull request merged**